www/templates/menu/profile.html

```diff
@@ -1,5 +1,47 @@
-<ion-view view-title="User Profile">
+<ion-view view-title="About You">
   <ion-content>
-    <h1>Profile</h1>
+    <div class="row">
+      <h2>Personal Info</h2>
+    </div>
+    <div class="row">
+      <div class="col col-25" style="height:100%">
+        <img src="http://www.bbsgphotography.com/images/commercial/headshots/2.jpg" class="item-image item-avatar" style="width:100%" alt="">
```

Comment:
refrain from using image urls directly from the internet, download them and add them to your assets better... you don't know when it may break